packages/app/app/layout.tsx

```diff
@@ -42,13 +45,22 @@ export const metadata: Metadata = {
   },
 };
 
+const siteId = process.env.NEXT_PUBLIC_FATHOM_SITE_ID;
+
 export default function RootLayout({ children }: PropsWithChildren) {
   return (
     <html lang="en" className={stabilGrotesk.variable}>
       <body className="font-sans bg-fixed bg-surface-25 bg-auto-100 bg-matrix-and-green-gradient text-em-high">
+        <Script
+          src="https://cdn.usefathom.com/script.js"
+          data-auto
+          data-site={siteId}
+          data-spa="hash"
+          defer
+        />
```

Comment:
Having this script here seems odd.

Reply:
Not sure if preferred the `loadFathom` code.  Do you know if works the same?

Reply:
All the third-party scripts we load (at least in Nextjs apps) should be preferred to be done with the `<Script />` component from Nextjs unless we have some other way to load/config them (like Tailwind).

The component also provides a few other props like "onLoad" or "onError" to trigger stuff as needed. Same goes for HTML `<head />` vs Nextjs `<Head />`.

These are optimized components to work with next, provided by them. 
Besides it's easier to read and identify IMHO, and simplifies the code as we load FA in a single place and do not use a JS function to manually pick, create, and modify a script tag. We were doing this because we don't have an actual HTML file to update on Next.

Reply:
Thanks for the explanation. I think we can go with this.